crates/ruff/src/rules/pep8_naming/rules/camelcase_imported_as_acronym.rs

```diff
@@ -0,0 +1,77 @@
+use ruff_macros::{define_violation, derive_message_formats};
+use ruff_python::string::{self};
+use rustpython_parser::ast::Stmt;
+
+use crate::ast::helpers::identifier_range;
+use crate::registry::Diagnostic;
+use crate::rules::pep8_naming::helpers;
+use crate::source_code::Locator;
+use crate::violation::Violation;
+
+define_violation!(
+    /// ## What it does
+    ///
+    /// Checks for CamelCase imports that are aliased as acronym.
+    ///
+    /// ## Why is this bad?
+    ///
+    /// [PEP8](https://peps.python.org/pep-0008/) recommends naming conventions for class names,
+    /// function names, constants etc. Inconsistenct [naming styles](https://peps.python.org/pep-0008/#descriptive-naming-styles)
```

Comment:
I think it would look better to use CommonMark link definitions i.e. only have `[PEP8]` here and under references have:

```
[PEP8]: https://peps.python.org/pep-0008/
```

(The reference heading can be removed if it only contains such definitions)